redhat-cop/containers-quickstarts
# Fedora based Asciidoctor

#### What is this PR About?
Describe the contents of the PR
Actually, a copy of ubi8-asciidoctor but now Fedora (fc) based.
This image is easier to maintain because version dependencies are handled by Fedora team

#### How do we test this?
Provide commands/steps to test this PR.
Create a pdf
And check this pdf with pdf generated by ubi8-asciidoctor
That worked out fine.
Newer versions can have another test cycle.

cc: @redhat-cop/day-in-the-life
